src/springfield/fields.py

```diff
@@ -390,12 +390,26 @@ def __init__(self, entity, *args, **kwargs):
                        Use 'self' to use the :class:`Entity` class that
                        this field is already bound to.
         """
-        self.type = entity
+        self._type = entity
         super(EntityField, self).__init__(*args, **kwargs)
 
+    @property
+    def type(self):
```

Comment:
If there's a better place for this, let me know. If not I'll update this with a docstring elaborating on why it's here instead of `init()` or `__init__()` due to the metaclass thing.